test/Mono.Linker.Tests.Cases/Libraries/CopyUsedAssemblyWithMainEntryRoot.cs

```diff
@@ -9,11 +10,16 @@ namespace Mono.Linker.Tests.Cases.Libraries
 	[Kept]
 	[KeptMember (".ctor()")]
 	[SetupLinkerAction ("copyused", "test")]
+	[SetupCompileBefore ("lib.dll", new[] { "Dependencies/CopyUsedAssemblyWithMainEntryRoot_Lib.cs" })]
+	[KeptMemberInAssembly ("lib.dll", typeof (CopyUsedAssemblyWithMainEntryRoot_Lib), "Used()")]
+	// Marked CopyUsed assemblies are not fully marked like Copy assemblies, so the Unused dependency is not kept.
+	[RemovedMemberInAssembly ("lib.dll", typeof (CopyUsedAssemblyWithMainEntryRoot_Lib), "Unused()")]
```

Comment:
This looks wrong, copyused should copy whole assembly. the only difference is that it will copy when anything from the assembly is used

Reply:
Agreed - I was surprised by this too - but I'm not trying to fix that here. I filed https://github.com/mono/linker/issues/1777 for it.